tests/unit/templates.test.ts

```diff
@@ -31,6 +32,14 @@ Deno.test("getConfigTemplate standard", () => {
 //     db: "nessie",
 // });
 
+// const client = new ClientMySQL55({
+//     hostname: "localhost",
+//     port: 3306,
+//     username: "root",
+//     // password: "pwd", // uncomment this line for <8
+//     db: "nessie",
+// });
+
```

Comment:
Let's not have the `ClientMySQL55` available as default. I rather have it as a notice in the readme and add it to the class comment of `ClientMySQL`.